There is a missing PyYAML dependency that can result in import error.

This was reported upstream:
https://bugs.launchpad.net/oslo.utils/+bug/1977850

--- oslo.utils-4.10.0/requirements.txt
+++ oslo.utils-4.10.0/requirements.txt
@@ -16,3 +16,4 @@ netifaces>=0.10.4 # MIT
 debtcollector>=1.2.0 # Apache-2.0
 pyparsing>=2.1.0 # MIT
 packaging>=20.4 # BSD
+PyYAML
